app/models/metric/statistic.rb

```diff
@@ -1,15 +1,18 @@
 module Metric::Statistic
   def self.calculate_stat_columns(obj, timestamp)
-    return {} unless obj.respond_to?(:container_groups)
-    return {} unless obj.respond_to?(:container_images)
-
+    hash = {}
```

Comment:
Maybe rename `hash` to `stats` or something less generic ? 
It is confusing, to me at least :-)